settings.gradle

```diff
@@ -0,0 +1 @@
+rootProject.name = 'opendistro-ism'
```

Comment:
shall we use a more self-explain name?

Reply:
also add license header in this file

Reply:
Added more descriptive name and copyright